agent/handlers/v1_handlers.go

```diff
@@ -92,6 +95,41 @@ func newTaskResponse(task *api.Task, containerMap map[string]*api.DockerContaine
 	}
 }
 
+func newContainerResponse(dockerContainer *api.DockerContainer, eni *api.ENI) v1.ContainerResponse {
+	container := dockerContainer.Container
+	resp := v1.ContainerResponse{
+		Name:       container.Name,
+		DockerId:   dockerContainer.DockerID,
+		DockerName: dockerContainer.DockerName,
+	}
+
+	for _, binding := range container.Ports {
```

Comment:
Can you add a getter in the container struct to return the Ports and use the getter instead?

Reply:
I'll make the change. Just wondering why we should use a getter if `Ports` is an exported field. I think we make it an exported field so we can serialize it.

Reply:
yes, some other fields like `KnowStatus` is exported for the same purpose.